peps/pep-0723.rst

```diff
@@ -101,14 +102,37 @@ This PEP defines a metadata comment block format loosely inspired [2]_ by
 
 __ https://docutils.sourceforge.io/docs/ref/rst/directives.html
 
-Any Python script may have top-level comment blocks that start with the line
-``# /// TYPE`` where ``TYPE`` determines how to process the content, and ends
-with the line ``# ///``. Every line between these two lines MUST be a comment
-starting with ``#``. If there are characters after the ``#`` then the first
-character MUST be a space. The embedded content is formed by taking away the
-first two characters of each line if the second character is a space, otherwise
-just the first character (which means the line consists of only a single
-``#``).
+Any Python script may have top-level comment blocks that MUST start with the
+line ``# /// TYPE`` where ``TYPE`` determines how to process the content. That
+is: a single number sign, followed by a single space, followed by three
```

Comment:
Consider changing instances of "a single number sign" to "a ``#`` sign"

This would improve readability and minimize confusion where "number sign" is not the preferred term for the symbol `#`

Reply:
Done, thanks!